### PR TITLE
Improved UI and accessbility

### PR DESCRIPTION
**Pull request by fozy81**
Made a few small changes to UI to pass Chrome lighthouse accessibility tests and general small tweaks to UI to improve look/feel. Hopefully be useful.

### CODE REVIEW

File: public/sw.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // cache, then increment the CACHE_VERSION value. It will kick off the ServiceWorker update
 // flow and the old cache will be purged as part of the activate event handler when the
 // updated ServiceWorker is activated
-var CACHE_VERSION = 2
+var CACHE_VERSION = 1
```
Comment (gr2m) [inline public/sw.js]: why not `3`? :)

File: test/walkthrough-test.js [hunk 2]
```diff
@@ -1,50 +0,0 @@
...
-        tt.end()
-      })
-
-      // cleanup
-        .localStorage('DELETE')
-    })
-
-    t.end()
-  })
-}
```
Comment (gr2m) [inline test/walkthrough-test.js]: did you intentionally remove all these tests? I guess they are obsolete, just want to make sure

File: public/sw.js (hunk 1 above)
Comment (fozy81) [reply public/sw.js]: I find I need to change the version during testing to refresh the cache in the browser ...easy to miss change there.

File: test/walkthrough-test.js (hunk 2 above)
Comment (fozy81) [reply test/walkthrough-test.js]: Yes, I removed the tests - I think there are all redundant after switching to cypress?

File: public/sw.js (hunk 1 above)
Comment (gr2m) [reply public/sw.js]: I just wondered why you decreased the number when the comment above says to increase it, that’s all :)

File: test/walkthrough-test.js (hunk 2 above)
Comment (gr2m) [reply test/walkthrough-test.js]: Yes, I thought so, too

File: public/sw.js (hunk 1 above)
Comment (fozy81) [reply public/sw.js]: I can change back in another commit?

**Comment by gr2m** [comment]
just a heads up, we use lowercase commis message prefixes. `Feat: Improve UI and accessibility` might not be recognized, better to use lowercase `feat: ...` in future

**Pull request merged**